build.gradle.kts

```diff
@@ -48,43 +48,7 @@ allprojects {
     mavenCentral()
     google()
   }
-}
-
-subprojects {
-  apply(plugin = "com.diffplug.spotless")
-  configure<SpotlessExtension> {
-    kotlin {
-      target("**/*.kt")
-      ktlint(versions.ktlint).userData(mapOf("indent_size" to "2"))
-      trimTrailingWhitespace()
-      endWithNewline()
-    }
-  }
-
-  tasks.withType<KotlinCompile>().configureEach {
-    kotlinOptions {
-      jvmTarget = JavaVersion.VERSION_1_8.toString()
-      @Suppress("SuspiciousCollectionReassignment")
-      freeCompilerArgs += "-Xjvm-default=all"
-    }
-  }
-
-  tasks.withType<JavaCompile> {
-    options.encoding = StandardCharsets.UTF_8.toString()
-    sourceCompatibility = JavaVersion.VERSION_1_8.toString()
-    targetCompatibility = JavaVersion.VERSION_1_8.toString()
-  }
-
-  tasks.withType<Test> {
-    testLogging {
-      events(STARTED, PASSED, SKIPPED, FAILED)
-      exceptionFormat = TestExceptionFormat.FULL
-      showStandardStreams = false
-    }
-  }
-}
 
-allprojects {
```

Comment:
Does it make any difference that this `allprojects` is at the top level of the script, while the one you merged it into is inside the `buildscript` block?

Reply:
`allprojects` is still at the top level.

![image](https://user-images.githubusercontent.com/10363352/132785906-6f291050-6af0-4939-86ad-0ecdcda20eb3.png)

Reply:
Got it, sorry, I misread the diff!

Reply:
Don't worry.